src/config/cmd.rs

```diff
@@ -26,9 +26,6 @@ fn merge_json(a: &mut Value, b: Value) {
         (Value::Array(ref mut a), Value::Object(b)) => {
             a.extend([Value::Object(b)]);
         }
```

Comment:
This block strikes me as a bit weird too. Maybe it should also just map to `b`? Or throw an error.

Reply:
@leebenson was there a specific case this was covering? Otherwise, happy to update this.

Reply:
Looking at this again, I think this can be dropped. Arrays should be overridden rather than combined, and I can't think why Value::Object was added as an exception.